libraries/AP_Baro/AP_Baro.h

```diff
@@ -16,6 +17,13 @@
 #define BARO_TIMEOUT_MS                 500     // timeout in ms since last successful read
 #define BARO_DATA_CHANGE_TIMEOUT_MS     2000    // timeout in ms since last successful read that involved temperature of pressure changing
 
+// maximum vertical accuracy from GPS that will be accepted as baro correction (meters)
+#define BARO_MAX_GPS_ACCURACY 2.0f
```

Comment:
This is very tight considering it is beyond the spec for most uncorrected L1 only chips.

Reply:
Strange. 3 times more than a simple without any correction ublox gives.